airflow/models.py

```diff
@@ -667,6 +667,7 @@ class Connection(Base, LoggingMixin):
         ('azure_data_lake', 'Azure Data Lake'),
         ('cassandra', 'Cassandra',),
         ('qubole', 'Qubole'),
+        ('cloudsql', 'Google Cloud SQL',),
```

Comment:
`cloudsql` -> `gcp_ cloudsql`

Reply:
Yep. That's fine, but it also implies changing cloudsql:// to gcp_cloudsql:// in URLs.  Which is good :).

Reply:
Unfortunately urllib.parse does not like '_' in scheme so I had to change it to gcpcloudsql:// instead. But I think that it is still fine this way